src/leap/keymanager/keys.py

```diff
@@ -120,7 +120,7 @@ def build_key_from_dict(kClass, kdict):
     :rtype: C{kClass}
     """
     try:
-        validation = toValidationLevel(kdict[KEY_VALIDATION_KEY])
+        validation = ValidationLevel(kdict[KEY_VALIDATION_KEY])
```

Comment:
I think I'd prefer the use like:

```
ValidationLevels.get(kdict[KEY_VALIDATION_KEY]))
```
where validation levels is a module instance